src/modules/ReduxTypes.js

```diff
@@ -69,16 +69,9 @@ export type State = {
         transaction: AbcTransaction | null,
         parsedUri: AbcParsedUri,
         error: Error | null,
-        displayAmount: number,
-        publicAddress: string,
-        feeSatoshi: number,
         label: string,
-        feeSetting: 'low' | 'standard' | 'high' | 'custom',
-        inputCurrencySelected: string,
-        maxSatoshi: number,
-        isPinEnabled: boolean,
-        isSliderLocked: boolean,
-        draftStatus: 'over' | 'under',
+        networkFeeOption: 'low' | 'standard' | 'high' | 'custom',
```

Comment:
You should declare a separate 

`export type FeeOptions = 'low' | 'standard' | 'high' | 'custom'`

Then use that type when you declare other variables that represent the `networkFeeOption` such as 

`updateMiningFee(networkFeeOption: FeeOptions)`